index.html

```diff
@@ -1387,9 +1379,44 @@ <h4 class="accordion-title font-size-19">
                       >
                         <div class="accordion-content">
                           <p>
-                            Bitcoin is designed around the idea of using
-                            cryptography to control the creation and transfer of
-                            money in any time.
+                            GIT sessions
+                          </p>
+                          <p>
+                            Competitive Programming Sessions
+                          </p>
+                          <p>
+                            Developer Series
+                          </p>
+                          <p>
+                            Workshops with the support of Industry
+                          </p>
+                          <p>
+                            Hackerthon Events
+                          </p>
+                          <p>
+                            Competitive Programming Sessions
+                          </p>
+                        </div>
+                      </div>
+                    </div>
+                    <div class="accordion-item panel">
+                      <div class="accordion-heading" role="tab" id="crypto-accordion-heading-4">
+                        <h4 class="accordion-title font-size-19">
+                          <a class="collapsed" data-toggle="collapse" data-parent="#crypto-accordion-1" href="#crypto-accordion-collapse-3"
+                            aria-expanded="false" aria-controls="crypto-accordion-collapse-3">
+                            How to reach us
```

Comment:
How to reach us not working. It displays the previous answers. @IsuruLakshan1997

Reply:
resolved